tests/test_edtfdatestring.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016-2020 CERN.
```

Comment:
This started 2020 AFAIK

Reply:
Just made the change
`# Copyright (C) 2020 CERN.`